scripts/idl/matter_idl_types.py

```diff
@@ -21,9 +21,14 @@ def __init__(self, meta: Meta = None, line: int = None, column: int = None):
             self.column = column
 
 
+class StructAttribute(enum.Enum):
+    FABRIC_SCOPED = enum.auto()
+
+
 class FieldAttribute(enum.Enum):
     OPTIONAL = enum.auto()
     NULLABLE = enum.auto()
+    FABRIC_SENSITIVE = enum.auto()
```

Comment:
I believe this would work if we wanted some binary serialization. However if I implemented serialization I think we probably would want text serialization instead  and say `NULLABLE` instead of `2`.